src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/ReportRunRepository.kt

```diff
@@ -101,4 +113,30 @@ class OrderlyWebReportRunRepository : ReportRunRepository
             return result.fetchInto(ReportRunWithDate::class.java)
         }
     }
+
+    override fun updateReportRun(
+        key: String,
+        status: String,
+        version: String?,
+        logs: List<String>?
+    )
+    {
+        val logsString = logs?.joinToString(separator = "\n")
+        val reportVersion = if (status == SUCCESS_STATUS)
```

Comment:
Although report version is included in status response from Orderly Server as soon as it starts running, the version will not exist in the database until it has successfully completed, so we cannot add it to the table until them (as it is a foreign key)